jujugui/static/gui/src/app/components/header-breadcrumb/_header-breadcrumb.scss

```diff
@@ -61,6 +61,7 @@
     line-height: $breadcrumb-line-height;
     margin-left: 20px;
     position: absolute;
+    min-width: 300px;
```

Comment:
Driveby ish....I couldn't reproduce this in the original app.js but in the new one, when rendering it would be like:

Loading
Model...

Then switch to:

Loading Model...

So my guess is that the explicit rendering order is causing the container to not be full width, whereas before we were relying on some implicit rendering to make the container full width.